http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -528,9 +534,19 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       apps <- listApp(
         userInfo,
         Some(cloudContext),
-        Map.empty
+        Map(includeDeletedKey -> "true")
+      )
+      _ <- apps.traverse(app =>
+        deleteAppRecords(userInfo, cloudContext, app.appName).handleErrorWith { err =>
```

Comment:
just to clarify, this is catching the case where Sam returns an error when checking permission for an app that doesn't exist?

Reply:
Yeah!